include/capstone.h

```diff
@@ -118,7 +120,8 @@ typedef struct cs_opt_mem {
 
 // Runtime option for the disassembled engine
 typedef enum cs_opt_type {
-	CS_OPT_SYNTAX = 1,	// Assembly output syntax
+	CS_OPT_NONE = 0,	// No opetion specified
```

Comment:
i am still looking at this PR. sorry for taking this too long, but too many things coming up now